### PR TITLE
Add stateful PlayScreen guide

### PR DESCRIPTION
**Pull request by luizgrp**
#### WHAT

Add stateful `PlayScreen` guide to docs.

#### Checklist :clipboard:
- [N/A] Add explicit visibility modifier and explicit return types for public declarations
- [N/A] Run spotless check
- [N/A] Run tests
- [N/A] Update metalava's signature text files

### CODE REVIEW

File: docs/media-playscreen.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Stateful PlayScreen guide
+
+This is a guide on how to use the stateful `PlayScreen` with your own implementation of `PlayerRepository`.
+
+## Basic usage
+
+1. Implement PlayerRepository
```
Comment (yschimke) [inline docs/media-playscreen.md:7]: Maybe describe what is required here typically.

For example you could call out Media3 Player, and how state could be mapped via a listener and a MutableState field?

Operations would be called on the player.

Maybe show implementations of two of these?

```
    var isPlaying = MutableStateFlow(false)

    private val listener = object : Player.Listener {
        override fun onIsPlayingChanged(isPlaying: Boolean) {
            updatePlayingState()
        }
    }
    
    player.addListener(listener)

    fun setPlaybackSpeed(speed: Float) {
        player.setPlaybackSpeed(speed)
    }
```

**Review by yschimke — APPROVED** [review]
Looks great!

File: docs/media-playscreen.md (hunk 1 above)
Comment (luizgrp) [reply docs/media-playscreen.md:7]: cool, I will add it in a separate PR 👍

Comment (luizgrp) [reply docs/media-playscreen.md:7]: done in https://github.com/google/horologist/pull/134

**Pull request merged**